openapi/platform/platform.yaml

```diff
@@ -3061,13 +3061,16 @@ paths:
                 deactivationType:
                   type: string
                   description: |-
-                    `cancel` cancel indicates we should turn the product or service off at the anniversary date, or commitment date, whichever is later and till then it will be available for use.
+                    `cancel` indicates we should turn the product or service off at the anniversary date, or commitment date, whichever is later and till then it will be available for use.
 
-                    `immediate` Immediate indicates we should turn the product or service off immediately and it will no longer be available to use.
+                    `immediate` indicates we should turn the product or service off immediately and it will no longer be available to use.
+                    
+                    `restore` indicates that we should undo the cancellation.
                   default: cancel
                   enum:
                     - cancel
                     - immediate
+                    - restore
```

Comment:
Skimming the PRs it looks like it would meet the partner's needs. 
The endpoint being modified is specifically a request cancelation action ( `/subscriptionAssignments/{id}/actions/requestCancellation` )

I wonder if it should be a new endpoint instead. Something like `/subscriptionAssignments/{id}/actions/undoCancellation` with no body params.

Reply:
Yeah, I was originally thinking that too. Was hoping I could cheat not doing the hard work but will see if thats feasible.